projects/ngrx.io/content/guide/data/entity-actions.md

```diff
@@ -17,19 +17,33 @@ It has additional properties that guide NgRx Data's handling of the action. Here
 <code-example header="EntityAction" linenums="false">
 export interface EntityAction&lt;P = any&gt; extends Action {
   readonly type: string;
+  readonly payload: EntityActionPayload&lt;P&gt;;
+}
+</code-example>
+
+<code-example header="EntityActionPayload" linenums="false">
+export interface EntityActionPayload&lt;P = any&gt; extends EntityActionOptions {
   readonly entityName: string;
-  readonly op: EntityOp;
-  readonly payload?: P;
+  readonly entityOp: EntityOp;
+  readonly data?: P;
+
+  // EntityActionOptions (also an interface)
+  readonly correlationId?: any;
+  readonly isOptimistic?: boolean;
+  readonly mergeStrategy?: MergeStrategy;
   readonly tag?: string;
   error?: Error;
   skip?: boolean
 }
 </code-example>
 
-* `type` - action name, typically generated from the `tag` and the `op`
-* `entityName` - the name of the entity type
-* `op` - the name of an entity operation
-* `payload?` - the message data for the action.
+* `type` - action name, typically generated from the `tag` and the `op`.
+* `entityName` - the name of the entity type.
+* `op` - the name of an entity operation.
+* `data?` - the message data for the action.
+* `correlationId?` - a serializable object (typically a string) for correlating related actions.
+* `isOptimistic?` - true if should perform the action optimistically (before the server responds).
+* `mergeStrategy` - How to merge an entity into the cache. See [Change Tracking](guide/data/entity-change-tracker).
```

Comment:
Consistency, all other sentences start in lowercase 
 
```suggestion
* `mergeStrategy` - how to merge an entity into the cache. See [Change Tracking](guide/data/entity-change-tracker).
```

Reply:
Done